phpBB/phpbb/install/module/obtain_data/task/obtain_admin_data.php

```diff
@@ -215,4 +219,10 @@ public function get_task_lang_name()
 	{
 		return '';
 	}
+
+	// Returns the task name for navigation bar
```

Comment:
Please add a proper docblock as employed by other methods. It should state what it does and what it's returning in this case.